tests/functional/strategy/test_migration.py

```diff
@@ -1,5 +1,7 @@
 import brownie
 
+ZERO_ADDRESS = "0x0000000000000000000000000000000000000000"
```

Comment:
I think Brownie actually defines this constant somewhere

Reply:
oh nice, will check if i can pull it from that

Reply:
done https://github.com/iearn-finance/yearn-vaults/pull/145/commits/62afcdcfe52be1dd2492853e29c84cf5d425e53d